javalib/src/main/scala/java/net/AbstractPlainSocketImpl.scala

```diff
@@ -1,10 +1,9 @@
 package java.net
```

Comment:
At first pass, using `InetAddress.getByAddress(sockaddrToByteArray(sa))` seems like an obvious 
simplification.    I am not certain but I believe that the `InetAddress.getByName(fromCString(ipstr))`
donkeying around is to allow IPv6 scope-ids to be handled (set in the InetAddress). If that is true,
it would have been nice if I had left a comment. Oops!

Unless there is a test case, I suggest leaving this file untouched and focusing on UDP specific issues.